compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -3,18 +3,22 @@ package tools
 package dotc
 
 import org.junit.Test
-import java.io.{ File => JFile }
-import org.junit.experimental.categories.Category
 
 import scala.util.matching.Regex
+import scala.concurrent.duration._
 
-@Category(Array(classOf[ParallelTesting]))
-class CompilationTests extends ParallelSummaryReport with ParallelTesting {
+import vulpix.{ ParallelTesting, SummaryReport }
+
+class CompilationTests extends SummaryReport with ParallelTesting {
   import CompilationTests._
 
-  def isInteractive: Boolean = ParallelSummaryReport.isInteractive
+  // Test suite configuration --------------------------------------------------
 
-  def testFilter: Option[Regex] = sys.props.get("dotty.partest.filter").map(r => new Regex(r))
+  def maxDuration = 180.seconds
```

Comment:
Note - we have some tests that run for quite a long time. In most cases these are benchmarks. But there are some which aren't and run for up to 60 seconds.

I propose to have a split between these run tests and the regular run tests in:

`./tests/run-long/``

And have an interface for them which allows for longer execution.